tests/pytest/test_fixtures.py

```diff
@@ -28,6 +32,26 @@ def infinity():
     return float('inf')
 
```

Comment:
It is perhaps gratuitous to have 4 fixtures and tests when the bug could really be reproduced with just 1.  I can change this if you like.